browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchLauncher.java

```diff
@@ -0,0 +1,92 @@
+package com.braintreepayments.api;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.fragment.app.FragmentActivity;
+
+import org.json.JSONObject;
+
+public class BrowserSwitchLauncher {
```

Comment:
```suggestion
/**
 * Class that manages the logic for browser switching.
 */
public class BrowserSwitchLauncher {
```

Not sure exactly what but should have a docstring